### PR TITLE
Update att desc

### PR DESCRIPTION
**Pull request by tylerrowsell**
Closes https://github.com/Shopify/custom-data-issues/issues/2851

The last update of the missing attribute descriptions and their localizations.

@elsom25 just to make sure I got this right... everything on `main` dists should be unstable right? I think this was a mistake I made when releasing `2024-07`

### CODE REVIEW

File: missing_attributes.csv
Comment (KComrade53) [inline missing_attributes.csv:1]: Did you mean to commit this?

File: script.rb
Comment (KComrade53) [inline script.rb:1]: Did you mean to commit this script?

**Comment by elsom25** [comment]
>everything on main dists should be unstable right?

Yep generally. We "tag" the moment that a release exists and main continues forward

File: VERSION [hunk 1]
```diff
@@ -1 +1 @@
-2024-07
+unstable
```
[the remark this answers is not in the document]
Comment (elsom25) [reply VERSION:1]: Or maybe `2024-07-unstable`? Or we keep it as raw `unstable`

**Pull request merged**